rtf_vis_tool/src/Components/AllFrustums.js

```diff
@@ -98,8 +99,8 @@ function AllFrustums() {
             const rotation_matrix = q.toMatrix(true);
             const rotation = nj.array(rotation_matrix);
             const translation = nj.array([tx,ty,tz]);
-            const wTc = new SE3(rotation, translation);
-            var verts_worldframe = frustumObj.get_mesh_vertices_worldframe(wTc);
```

Comment:
let's keep it the way I had in my PR: https://github.com/borglab/gtsfm/pull/297/files

All we need to do is add the inverse operation to the SE(3) class, and call it here.